### PR TITLE
Guild Toys

### PR DESCRIPTION
**Pull request by Lamasmaster**
## About The Pull Request
<details>
<summary>
	Adds what is essentially AMR at home kind of weapon in Elefant and another tier to the katana being a vibrokatana
Both of these are guild only and are meant as a way to spend resources guild normally doesn't touch in any way (platinum and uranium) as well as giving them finally a handcrafted ballistic weapon.

E3
E4

E2
E1

</summary>
<hr>

<hr>
</details>

## Changelog
:cl:
add: Two new weapons for guild, Elefant and Vibrokatana
/:cl:

### CODE REVIEW

File: code/modules/projectiles/guns/projectile/boltgun/elefant.dm [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+	caliber = CAL_ANTIM
+	recoil_buildup = 70
+	handle_casings = HOLD_CASINGS
+	load_method = SINGLE_CASING
+	max_shells = 3
+	ammo_type = /obj/item/ammo_casing/antim
+	fire_sound = 'sound/weapons/railgun.ogg'
+	reload_sound = 'sound/weapons/guns/interact/rifle_load.ogg'
+	matter = list(MATERIAL_PLASTEEL = 40, MATERIAL_STEEL = 20, MATERIAL_PLASTIC = 10)
+	price_tag = 10000
```
Comment (Trilbyspaceclone) [inline code/modules/projectiles/guns/projectile/boltgun/elefant.dm:20]: Why so much?

Comment (r4d6) [reply code/modules/projectiles/guns/projectile/boltgun/elefant.dm:20]: It's something you make by hand, not printed. So it make sense that recycling it give you back **most** of what you used to make it.

Comment (Trilbyspaceclone) [reply code/modules/projectiles/guns/projectile/boltgun/elefant.dm:20]: the price tag

**Comment by Trilbyspaceclone** [comment]
whats the point of the new katana?

File: code/modules/projectiles/guns/projectile/boltgun/elefant.dm (hunk 1 above)
Comment (Lamasmaster) [reply code/modules/projectiles/guns/projectile/boltgun/elefant.dm:20]: Whoops I forgot to change that over

**Comment by Lamasmaster** [comment]
To have a viable melee option since the actual katana isn't any better than a chainsaw or even machete/claymore and considering it costs diamonds to make unlike those the payoff is minimal. 30 AP is just about enough to kill throught riot security armor in about 8 hits to the head with base robustness (0) and without mods with how I tested it so it's more PVP than PVE weapon.

**Comment by Kazkin** [comment]
The stats and  mechanics look good, but there's a couple design issues that need to be fixed.

1. Change the name of the laser katana to not be directly related to metal gear rising.
2. The color scheme for the elefant weapon is the marshal colors, which... is odd, it should have a similar color scheme to the reductor, myr, or abdicator. Weapons like the bison have that color because they are a mod of the existing marshal weapon, the bull.
3. The naming scheme of the elefant also breaks convention, as a guild weapon it should use the same naming convention. I suggest "auxilia" since its meant to be an alternate/sub type of the anti-material rifle.
4. How is a pneumatic gun made from a grenade launcher being converted into a bolt action rifle that fire .60 and then ejects a shell casing? Wouldn't it be firing the entire bullet if it was pneumatic?

Other than the visual design issues, the mechanics are pretty solid, so well done there.

**Comment by Lamasmaster** [comment]
Katana changed to just artificer katana and removed the description reference
Changed color scheme to abdictor
Changed name from Elefant to Venator (latin for hunter), if that is really how guild patterns their stuff from what I gathered
It doesn't it was mostly WIP I suppose, now it fires spikes in a way as lodge weapons do, had to adjust modifier for it to do around same damage. So it solves the last problem too

E7
E6

**Comment by Trilbyspaceclone** [comment]
Why are they now a crossbow caliber? this steps onto Hunters calibers/crafting a bit as they are meant to be the only ones with that kinda thing to keep their guns uniquic

File: code/game/objects/items/weapons/tools/simple_weapons.dm [hunk 2]
```diff
@@ -230,6 +230,18 @@
...
+	desc = "A modern unbranded katana sharpened and painted over with red all over it's blade. Just swinging it makes your ears ring from it's sharp edge cutting up against the air. The handle bears a thunderbolt logo."
+	icon_state = "katana_vibro"
+	item_state = "katana_vibro"
+	matter = list(MATERIAL_PLASTEEL = 10, MATERIAL_STEEL = 5, MATERIAL_DIAMOND = 2, MATERIAL_PLATINUM = 1, MATERIAL_URANIUM = 1)
+	hitsound = 'sound/weapons/anime_sword.wav'
+	tool_qualities = list(QUALITY_CUTTING = 35, QUALITY_SAWING = 10) // It can cut throught armor like nothing
+	slot_flags = SLOT_BELT | SLOT_BACK
+	force = WEAPON_FORCE_BRUTAL
+	armor_penetration = ARMOR_PEN_MASSIVE
+
```
Comment (Trilbyspaceclone) [inline code/game/objects/items/weapons/tools/simple_weapons.dm:244]:
```suggestion
	max_upgrades = 6 //Are main draw other then raw stats is are modality 

```

**Comment by r4d6** [comment]
> Why are they now a crossbow caliber? this steps onto Hunters calibers/crafting a bit as they are meant to be the only ones with that kinda thing to keep their guns uniquic

It's a crossbow caliber because I think that @Lamasmaster couldn't manage to make it fire regular bullets without having _those_ leave casings behind, which is pretty weird for a pneumatic gun.

**Comment by Lamasmaster** [comment]
I figured it made more sense for it to fire some sort of rod more than actual bullet which would also at same time give hunters a reason to interact with guild since they can buy these for their own weapons Not in any way meant to step on hunter's niche just don't know if making another bullet subtype would be a great idea since we have so many weapons already that just use their own snowflake ammo type.

E9

File: code/modules/projectiles/projectile/bullettypes.dm [hunk 3]
```diff
@@ -708,3 +708,14 @@
...
 	affective_ap_range = 9
 	nocap_structures = TRUE //Can do well againt walls and doors
+
+/obj/item/projectile/bullet/crossbow_bolt/railway
+	name = "railway spike"
+	icon_state = "SpearFlight"
+	damage_types = list(BRUTE = 55)
+	armor_penetration = 65 //Diamond Tipped
+	penetrating = 3
+	knockback = 1
```
Comment (Trilbyspaceclone) [inline code/modules/projectiles/projectile/bullettypes.dm:718]:
```suggestion
	//knockback = 1 stops an exploit
```

**Comment by Lamasmaster** [comment]
Going to remake this PR later considering my branch decided to commit sepuku and also because I wanted to add bit more.

**Pull request closed without merging**